gulpfile.babel.js

```diff
@@ -97,7 +97,7 @@ tasks.clean = async function clean() {
 
 tasks.lint = function lint() {
   return src(['*.js', 'src/**/*.js', 'scripts/**'])
-    .pipe(eslint())
+    .pipe(eslint({ fix: true }))
```

Comment:
How do you feel about this? 🤔 Made rule changing easier to carry out, but I'm not sure if we want to keep it.

Reply:
Never mind, this won't be ok for the CI/CD.